modules/machineset-yaml-aws.adoc

```diff
@@ -27,51 +33,61 @@ kind: MachineSet
 metadata:
   labels:
     machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
-ifndef::infra[]
+ifndef::infra,edge[]
   name: <infrastructure_id>-<role>-<zone> <2>
-endif::infra[]
+endif::infra,edge[]
 ifdef::infra[]
   name: <infrastructure_id>-infra-<zone> <2>
 endif::infra[]
+ifdef::edge[]
+  name: <infrastructure_id>-edge-<zone> <2>
+endif::edge[]
   namespace: openshift-machine-api
 spec:
   replicas: 1
   selector:
     matchLabels:
       machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
-ifndef::infra[]
+ifndef::infra,edge[]
       machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>-<zone> <2>
```

Comment:
No `machine.openshift.io/cluster-api-machineset: <infrastructure_id>-edge-<localzone_az_name>` label in this 
 sample manifest?

Reply:
Hi @mtulio and @yunjiang29 .

[edge] in this case refers to the post install AWS LZ doc?